test/StripeResource.spec.ts

```diff
@@ -262,4 +262,27 @@ describe('StripeResource', () => {
       ]);
     });
   });
+
+  describe('usage', () => {
```

Comment:
Do we have something that validates the actual header value? I'm worried that this test passing doesn't prove that usage is actually getting sent.

Reply:
Good idea. That's consistent with the other libraries too.

Reply:
Added!